deploy/sdk/src/dynamo/sdk/core/lib.py

```
@@ -85,35 +85,59 @@
+    
+    Args:
+        name: Optional name for the readiness probe route. Defaults to "/readyz".
+    """
+    def decorator(func: G) -> G:
+        if not callable(func):
+            raise TypeError("@readiness can only decorate callable methods")
+
+        func.__is_readiness_probe__ = True  # type: ignore
+        func.__readiness_route__ = name  # type: ignore
+        return func
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)